tests/integration/pilot/envoyfilter/main_test.go

```diff
@@ -75,6 +74,46 @@ spec:
             function envoy_on_response(handle)
               handle:logWarn("DEBUG RESPONSE")
             end
+  - applyTo: NETWORK_FILTER
+    match:
+      context: SIDECAR_INBOUND
+      listener:
+        portNumber: 80
+        filterChain:
+          filter:
+            name: "envoy.http_connection_manager"
+    patch:
+      operation: MERGE
+      value:
+        typed_config:
+          "@type": "type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager"
+          access_log:
+          - name: envoy.http_grpc_access_log
+            config: 
+              common_config:
+                log_name: "grpc-als-example"
+                grpc_service:
+                  envoy_grpc:
```

Comment:
It seems this does not work with the envoy_grpc client as it is required to use a static cluster for ALS (works for ext_authz). See: https://github.com/istio/istio/issues/16393#issuecomment-523609369 for a working example.

Reply:
Yes I am aware. But this is just a test, to ensure that changes deep in a HCM can be made.